demo/app/App.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { createPortal } from 'react-dom';
```

Comment:
please remove not used `useRef` and `createPortal`

Reply:
same, they're not being used in this file at all

Reply:
addressed here [3f12a29](https://github.com/hCaptcha/react-hcaptcha/pull/185/commits/3f12a29d517a2821b4ef5bad72bd075b544fe5c9)